BaseApi.js

```diff
@@ -217,7 +218,11 @@ const API = {
   getAuthenticatedUser(token = null) {
     const url = endpoints.account
     return makeGetRequest(url, token)
-  }
+  },
+  editUser(userPk, data, token = null) {
+    const url = endpoints.accounts.replace('$userPk', userPk)
+    return makePutRequest(url, data, token)
+  },
```

Comment:
getting a 400 would like to pair to understand why a bit more

Reply:
The account url doesnt need a user pk, it always operates on the logged in user. I made it that way to get the avatar on the ExplorePage, where we dont have the user pk yet..